dashboards/kuberay/ray-overview.json

```diff
@@ -825,12 +843,21 @@
                   "plotType": "LINE",
                   "targetAxis": "Y1",
                   "timeSeriesQuery": {
-                    "prometheusQuery": "sum(ray_node_disk_usage{cluster=~\"${Cluster.value}\",namespace=~\"${Namespace.value}\",location=~\"${Location.value}\",SessionName=~\"${SessionName.value}\"}) / on() (sum(ray_node_disk_free{cluster=~\"${Cluster.value}\",namespace=~\"${Namespace.value}\",location=~\"${Location.value}\",SessionName=~\"${SessionName.value}\"}) + sum(ray_node_disk_usage{cluster=~\"${Cluster.value}\",namespace=~\"${Namespace.value}\",location=~\"${Location.value}\",SessionName=~\"${SessionName.value}\"})) * 100",
-                    "unitOverride": ""
+                    "prometheusQuery": "sum(ray_node_disk_usage{cluster=~\"${Cluster.value}\",namespace=~\"${Namespace.value}\",location=~\"${Location.value}\",SessionName=~\"${SessionName.value}\"}) / (sum(ray_node_disk_free{cluster=~\"${Cluster.value}\",namespace=~\"${Namespace.value}\",location=~\"${Location.value}\",SessionName=~\"${SessionName.value}\"}) + sum(ray_node_disk_usage{cluster=~\"${Cluster.value}\",namespace=~\"${Namespace.value}\",location=~\"${Location.value}\",SessionName=~\"${SessionName.value}\"})) * 100",
+                    "unitOverride": "percent"
```

Comment:
All of the queries in this chart should be "unitOverride": "%" to display correctly
![Screenshot 2025-06-17 at 4 00 26 PM](https://github.com/user-attachments/assets/bf5b8341-bf36-4889-9e36-5f9fea39abb4)